css/about.css

```diff
@@ -0,0 +1,383 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@300;400&display=swap');
+
+@font-face {
+  font-family: 'COCOGOOSE';
+  font-style: normal;
+  font-weight: 400;
+  src: local('COCOGOOSE'), url('https://fonts.cdnfonts.com/s/26936/cocogoose_trial.woff') format('woff');
+}
+
+*,
+*::before,
+*::after {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+:root {
+  --color-black: #272a31;
+  --color-another-black: #575757;
+  --color-gray: #d3d3d3;
```

Comment:
The pages use the exact same colors, for text, backgrounds, borders, etc, from the original design 🔥 Well done!